app/views/licences/tabs/set-up.njk

```diff
@@ -80,3 +80,68 @@
     classes: "govuk-button--secondary"
   }) }}
 {% endif %}
+
+<hr />
+
+{% set agreementRows = [] %}
+
+{% set agreementHead = [
+  {
+    text: "Start date"
+  },
+  {
+    text: "End date"
+  },
+  {
+    text: "Agreement"
+  },
+  {
+    text: "Date signed"
+  },
+  {
+    text: "Action"
+  }
+] %}
+
+{% for agreement in  agreements %}
```

Comment:
can you move this logic into the presenter please? and just pass the array to the nunucks file?